### DISCUSSION TITLE
Emergency Access Compatibility with Bitwarden?

### DISCUSSION THREAD

**Discussion by Petruchio96** [dani-garcia/vaultwarden#3362] [Q&A] [answered]
I use Vaultwarden, but I'd like to have a relative who's a Bitwarden customer setup as one of my emergency access contacts. He would like to do the same with me. Is that possible?

**Comment by jjlin** [comment]
No, Bitwarden doesn't currently support federation, nor are there any plans to add that AFAIK. So you couldn't do this even if you were running the official self-hosted Bitwarden server.

**Comment by stefan0xC** [comment, marked as answer]
It's possible in the sense that you can invite him and he can invite you but then both of you will have to create a separate account for each platform, but I guess that is not what you had in mind?
